src/spec/intersection_spec.js

```diff
@@ -0,0 +1,18 @@
+import chai from 'chai'
+import intersection from '../lib/intersection.js'
+const expect = chai.expect
+
+describe('Intersection', () => {
+  it('Creates an array of unique values that are included in all given arrays', () => {
+    expect(intersection([4,2,7,5],[4,3,8,6])).to.eql([4])
+  })
+  it('Creates an array of unique values that are included in all given arrays', () => {
+    expect(intersection([6,10,7],[4,3,8,6])).to.eql([6])
+  })
+  it('Creates an array of unique values that are included in all given arrays', () => {
```

Comment:
This unit test should have a different description that is explicit regarding what it's testing. Perhaps something like `it( 'returns an empty array when there are no matching values' )`